README.md

```diff
@@ -2,6 +2,20 @@
 
 This repository is the home of all Tango Open Specification.
 
+## Mission
+
+The goal of this RFC project is to provide a formal specification of the current (v9.3 lts) Tango Controls system. This specification shall include:
```

Comment:
@andygotz and/or @bourtemb, I have updated this with the version number according to what we talk on Monday.
Let me know if it is ok, so I can merge this PR.

Reply:
I would just specify the version as "V9 LTS"

Reply:
@andygotz, thanks, done. Please, let me know if it is now ok.